cmd/dagger/call.go

```diff
@@ -55,6 +41,7 @@ dagger call lint stdout
 		case Terminal:
 			c.Select("websocketEndpoint")
 		default:
+			// FIXME: don't make this an error, it's confusing to users
```

Comment:
This error has been replaced 100% with Option 2 from (PR soon):
- https://github.com/dagger/dagger/issues/6515

Basically any chain that ends in an object that has itself functions, lists them under "Available Functions".